tools/clang/lib/SPIRV/SPIRVEmitter.h

```diff
@@ -320,8 +321,8 @@ class SPIRVEmitter : public ASTConsumer {
   ///
   /// The wrapper function is also responsible for initializing global static
   /// variables for some cases.
-  uint32_t emitEntryFunctionWrapper(const FunctionDecl *entryFunction,
-                                    uint32_t entryFuncId);
+  bool emitEntryFunctionWrapper(const FunctionDecl *entryFunction,
+                                uint32_t entryFuncId);
```

Comment:
This is an interesting feature of C/C++ I think. From the callers POV, it doesn't really matter whether `entryFuncId` (of POD type) is `const` or not since it's pass-by-value. But on the implementation side, `entryFuncId` is just like a function-scope variable. Putting `const` will make sure we don't write to the "variable".

So the compiler allows mismatch of `const`. :)